src/composables/LetoModelizerApi.js

```diff
@@ -0,0 +1,61 @@
+import axios from 'axios';
+
+/**
+ * Get the url to authenticate the user through its github provider.
+ * @returns {Promise<object>} Promise with the url on success
+ * otherwise the error that happened during the request.
+ */
+export async function getAuthenticationUrl() {
+  return axios.get('backend/authenticationUrl');
+}
+
+/**
+ * Log the user through the Parse-Server backend using the temporary code
+ * obtained from the github provider (from the url given by the backend).
+ * @param {string} temporaryCode The temporary code obtained from the github provider.
+ * @returns {Promise<object>} Promise with the logged-in user information on success
+ * otherwise the error that happened during the request.
+ */
+export async function login(temporaryCode) {
+  const headers = {
+    headers: {
+      Accept: 'application/json',
+      'X-Parse-Application-Id': process.env.BACKEND_APP_ID,
+    },
+  };
+  const params = {
+    code: temporaryCode,
+  };
+
+  return axios.post('/backend/api/functions/GitHubAuth', params, headers)
+    .then((response) => {
+      const firstName = response.data.result.firstname;
+      const userName = response.data.result.username;
+      const userParams = {
+        authData: response.data.result.authData,
+        firstname: firstName,
+        username: userName,
```

Comment:
why create variable for firstName and userName but not authData?

Reply:
It was some remaning code from when I was debugging, thanks ;)